ProcessMaker/Http/Resources/V1_1/CaseResource.php

```diff
@@ -28,10 +30,41 @@ public function toArray($request): array
     {
         $data = [];
 
+        $users = User::select('id', 'firstname', 'lastname', 'title', 'avatar')
```

Comment:
Plase use DB instead Eloquent because the eloquent model is 10 times slower.
Consider to include a filter by the participant user_ids present in the cases result.

Reply:
To improve the performance the `CaseResouce` and `CaseController` were refactored